### PR TITLE
Converted PrecertChainEntry to a POJO.

### PR DESCRIPTION
**Pull request by pphaneuf**

### CODE REVIEW

File: java/ct.proto [hunk 1]
```diff
@@ -51,16 +51,6 @@ message PreCert {
   optional bytes tbs_certificate = 2;
```
Comment (phad) [inline java/ct.proto]: I'm surprised you're not ditching this message at the same time, as it's so tightly coupled.

File: java/src/org/certificatetransparency/ctlog/PrecertChainEntry.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
+package org.certificatetransparency.ctlog;
+
+import org.certificatetransparency.ctlog.proto.Ct;
+
+
+public class PrecertChainEntry {
+  // The chain certifying the precertificate, as submitted by the CA.
+  public java.util.List<byte[]> precertificateChain;
```
Comment (phad) [inline java/src/org/certificatetransparency/ctlog/PrecertChainEntry.java]: import java.util.List and make this 'public List'

**Comment by phad** [comment]
Changes look OK but check what's up with Travis.

File: java/ct.proto (hunk 1 above)
Comment (pphaneuf) [inline java/ct.proto]: Coming right up in 4e11c1866676edc7cb2a270309dfcfddd697b3fd. :wink:

File: java/src/org/certificatetransparency/ctlog/PrecertChainEntry.java (hunk 2 above)
Comment (pphaneuf) [inline java/src/org/certificatetransparency/ctlog/PrecertChainEntry.java]: Done.

**Comment by pphaneuf** [comment]
Travis is hosed for us at the moment because of travis-ci/apt-source-whitelist#279... This is another case where separate repositories would help.

**Comment by pphaneuf** [comment]
(I did run `ant build test` by hand, and it passed)

**Comment by phad** [comment]
OK, LGTM and "LGTTravis" on its behalf.

**Pull request merged**